src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
+import core.basesyntax.model.Bulldozer;
+import core.basesyntax.model.Excavator;
+import core.basesyntax.model.Truck;
+
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
```

Comment:
While the use of polymorphism here is correct, ensure that the messages within the 'doWork' and 'stopWork' methods of the respective classes are informative and indicate which type of machine is working or stopping. The checklist emphasizes the importance of informative messages.